Riot/Modules/Settings/SettingsViewController.m

```diff
@@ -510,7 +511,9 @@ - (void)updateSections
     
     if (BuildSettings.roomScreenAllowTimelineStyleConfiguration)
     {
-        [sectionUserInterface addRowWithTag:USER_INTERFACE_TIMELINE_STYLE_INDEX];
+        // NOTE: Message bubbles are under labs section atm
+        
+//        [sectionUserInterface addRowWithTag:USER_INTERFACE_TIMELINE_STYLE_INDEX];
```

Comment:
Why keep this commented out code?

Reply:
We will remove lab flag in next releases and remove the associated lab section value. It's temporary.